client/state/data-layer/wpcom/comments/index.js

```diff
@@ -10,14 +10,41 @@ import { isDate } from 'lodash';
 import {
 	COMMENTS_REQUEST,
 	COMMENTS_RECEIVE,
-	COMMENTS_COUNT_RECEIVE
+	COMMENTS_REMOVE,
+  COMMENTS_COUNT_INCREMENT,
```

Comment:
thanks for catching this! Spaces vs tabs... how did it got past eslint? Changed on fab886cd6dcb8755059a844846e6ce1243bd6867.